src/absil/ilreflect.fs

```diff
@@ -1917,7 +1965,7 @@ let buildModuleTypePass4 visited   emEnv tdef = buildTypeDefPass4 visited [] emE
 // buildModuleFragment - only the types the fragment get written
 //----------------------------------------------------------------------------
     
-let buildModuleFragment cenv emEnv (asmB : AssemblyBuilder) (modB : ModuleBuilder) (m: ILModuleDef) =
+let buildModuleFragment cenv emEnv (_asmB : AssemblyBuilder) (modB : ModuleBuilder) (m: ILModuleDef) =
```

Comment:
The technique is to put a ``ignore asmB`` in the ``#if`` rather than to use an ``_asmB`` name.